docs/2.0-migration-guide.md

```diff
@@ -0,0 +1,78 @@
+---
+id: 2.0-migration-guide
+title: 2.0 Migration Guide
```

Comment:
We may want to change the name to something like "What's new in 2.0" as this doesn't come off as a guide, they're more like update docs!

Reply:
I see!